pkg/compiler/codegen.go

```diff
@@ -350,13 +351,18 @@ func (c *codegen) emitDefault(t types.Type) {
 // convertGlobals traverses the AST and only converts global declarations.
 // If we call this in convertFuncDecl, it will load all global variables
 // into the scope of the function.
-func (c *codegen) convertGlobals(f *ast.File, _ *types.Package) {
+func (c *codegen) convertGlobals(f *ast.File, pTypes *types.Package, globalVarsUsage funcUsage) {
 	ast.Inspect(f, func(node ast.Node) bool {
 		switch n := node.(type) {
 		case *ast.FuncDecl:
 			return false
 		case *ast.GenDecl:
-			ast.Walk(c, n)
+			// Walk only named used var or unnamed var that have function call inside.
+			if n.Tok == token.VAR {
+				ast.Walk(c, n)
+			} else {
+				ast.Walk(c, n)
```

Comment:
?

Reply:
It's a merge artifact, this code shouldn't be changed.